src/index.ts

```diff
@@ -220,7 +222,8 @@ function getPmtInf(
       );
     }
 
-    if (options?.checkBIC && !isValidBIC(item.bic)) {
+    const bicOptional = PAIN_TYPES[painFormat] === "CstmrCdtTrfInitn";
```

Comment:
```suggestion
    const bicOptional = PAIN_TYPES[painFormat] === "CstmrDrctDbtInitn";
```

Reply:
I don't think this is correct, because then pain.001.001.03 and pain.001.003.03 will support optinal bic and not pain.008.001.02 and pain.008.003.02

Reply:
Hi @jerebtw, pain.001... are for credit transfers, and pain.008 are for direct debit, which is where bic is optional. That's why I think that the bic optional variable shouod be true if pain type is CstmrDrctDbtInitn (which are pain.008...) and not CstmrCdtTrfInitn (which are pain.001...)

Reply:
To make it even easier, I think credit transfers also allow bic optional, as [this article](https://www.europeanpaymentscouncil.eu/what-we-do/sepa-payment-scheme-management/schemes-rely-global-open-standards) says.